development/decision_process.rst

```diff
@@ -0,0 +1,182 @@
+
+Change-controlled documents with other handles, such as LSE- or LPM-, including inter-subsystem interfaces, have project-wide change control processes.
+Please consult the DM PM, SA, or IS for more information.
+
+At least one member of the DM TCT_ will read each RFC to determine if it might affect a change-controlled document.
+
+.. _decision-making-rfc-responsibility:
+
+Responsibility and delegation
+-----------------------------
+
+For project management purposes, RFCs are formally proposals made to the DM PM and PS who by default are responsible for everything in DM (they "own" all problems).
+As owners, they have the final word in accepting or rejecting all proposals.
+Functionally, they delegate that ownership---the right and responsibility to make decisions---to others within the team (e.g. the SA, IS, group leads, etc.) who are expected to delegate it even further.
+Notifying your institutional technical manager about an RFC serves to inform the DM PM.
+
+.. _decision-making-rfd:
+
+Request for Discussion (RFD) Process
+====================================
+
+.. See RFC-53: https://jira.lsstcorp.org/browse/RFC-53
+
+Requests for Discussion (RFD) are intended to facilitate in-depth technical discussions across DM.
+These might be:
+
+- Detailed design discussions for a component of the system or its interfaces.
+- Design reviews for new code or refactorings of old code.
+- Brainstorming methods to solve difficult problems.
+- "Brain dump" explanations of a design to share knowledge across DM.
+
+.. _decision-making-rfd-creating:
+
+Creating an RFD
+---------------
+
+Create an RFD by making a JIRA issue in the `Request For Comments (RFC) <https://jira.lsstcorp.org/projects/RFC>`_ project with a **component of DM** and a **story type of RFD**.
+Use the **Location** field to specify the date and time of the discussion (:ref:`see below for time slot <decision-making-rfd-time>`).
+In the RFD's description:
+
+- Summarize the issue, and indicate a desired outcome from the discussions.
+- Include background material (using JIRA attachments, if necessary).
+- Provide a link to the BlueJeans or Google Hangouts room.
+
+Creating an RFD issue will trigger postings to the `dm-devel mailing list <https://lists.lsst.org/mailman/listinfo/dm-devel>`_ and the `'Bot: RFC' HipChat room <hipchat://hipchat.com/room/1028779>`_.
+
+As the discussion organizer, you are responsible for ensuring that key attendees are available for the time slot.
+DM members can comment on the RFD issue to indicate their availability, or whether the subject being discussed has already been resolved or covered elsewhere.
+
+.. _decision-making-rfd-time:
+
+The RFD time slot
+-----------------
+
+RFDs can happen weekly on Tuesdays from 12:30 to 2 PM Pacific.
```

Comment:
Should probably note that this is a slot that has been reserved for this purpose on most people's calendars and so has a greater probability of people being available.  But other times can be chosen if they work better for the required attendees.